shared/js/textChatStore.js

```diff
@@ -49,6 +49,9 @@ loop.store.TextChatStore = (function() {
       }
 
       this._dataDriver = options.dataDriver;
+
+      loop.subscribe("Panel:SetOwnDisplayName",
+                     this.updateOwnDisplayName.bind(this));
```

Comment:
the `participantStore`should also subscribe to this event, so the presence is updated on the sidebar when the name changes.